pkg/render/dex_test.go

```diff
@@ -354,6 +357,48 @@ var _ = Describe("dex rendering tests", func() {
 			Expect(deploy.Spec.Template.Spec.Affinity).To(Equal(podaffinity.NewPodAntiAffinity("tigera-dex", "tigera-dex")))
 		})
 
+		It("should render configuration with Resource requests and limits", func() {
```

Comment:
I think it would be good to cover the init container resources as well, since this is easy to break if we change the name schema for the container at some point.